src/components/ChatLog.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './ChatEntry.css';
+import './ChatLog.css';
+
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({ entries, updateLike }) => {
+  const messages = entries.map((message) => {
+    return (
+      <div key={message.id}>
+        <ChatEntry
+          id={message.id}
+          sender={message.sender}
+          body={message.body}
+          timeStamp={message.timeStamp}
+          liked={message.liked}
+          updateLike={updateLike}
+        />
+      </div>
+    );
+  });
+
+  return <section className="chat-log">{messages}</section>;
+};
+
+ChatLog.propTypes = {
+  entries: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number,
```

Comment:
I'd recommend making the `id` and `liked` props required since they are needed to make the like button work.